src/components/Availability.js

```diff
@@ -13,7 +15,17 @@ export default function Availability({ entry }) {
             }
         }
         if (!availableSlots.length && entry.hasAppointments) {
-            return <div>No date-specific data available.</div>;
+            return (
+                <div>
+                    No date-specific data available.
+                    <HelpDialog 
+                        title='No date-specific data available.' 
```

Comment:
I like the copy for this one! Could you run `prettier` on this file so that the formatting is consistent (things like ' vs ", for eaxmple)?